website/docs/guides/plurals.md

```diff
@@ -21,7 +21,7 @@ Good thing is that **as developers, we have to know only plural forms for the so
 
 If we use English in the source code, then we'll use only *one* and *other*:
 
-``` js
```

Comment:
Syntax highlighting in the intelij based IDEs doesn't work when there is a space.